docs/docsite/rst/dev_guide/developing_inventory.rst

```diff
@@ -61,6 +61,40 @@ When using the 'persistent' cache, inventory plugins can also use the configured
 
 .. _developing_an_inventory_plugin:
 
+Directory layout
+----------------
+
+The `PluginLoader` class defines a property named `config`, which is a list of directories that will be searched for inventory plugins. By default, it may contain::
+
+   /home/foo/.ansible/plugins/inventory
+   /usr/share/ansible/plugins/inventory
+   /usr/lib/python3.7/site-packages/ansible/plugins/inventory
+   /usr/lib/python3.7/site-packages/ansible/plugins/inventory/__pycache__
+
+This means that if you enable your `foo` dynamic inventory in `ansible.cfg` and save its `foo.py` in one of those directories, Ansible will try to use it while parsing your inventory file.
+
+Example::
+
+    /etc/ansible/ansible.cfg
+      [inventory]
+      enable_plugins = foo, host_list, script, auto, yaml, ini, toml
+
+      [defaults]
+      inventory = /home/foo/ansible/inventories
+
+    /home/foo/.ansible/plugins/inventory/foo.py
+      (see below)
+
+    /home/foo/ansible/.
```

Comment:
```suggestion
     # /home/foo/ansible/ directory
```

Reply:
the word `foo` is used in lot of places in the examples, `foo.yml`, `plugin: foo`, the home `foo`, I would replace plugin `foo` by `sample_plugin`, and /home/foo by `$HOME`.